src/main/resources/application.yml

```diff
@@ -100,6 +100,19 @@ features:
         update-prison-location: false
   apply-limited-access-markers: false
 
+new-nsi:
+  referral:
+    provider-code: CRS
+    staff-code: CRSUATU
+    team-code: CRSUAT
+    nsi-status: INPROG
+    referral-type-to-nsi-type: '{
```

Comment:
This could be in the standard YAML map format?

Reply:
Because of the spaces and special characters, the standard YAML format becomes a little unreadable.

Reply:
Found that if I contain the name in square brackets it's fine. Will change:

    service-category-to-nsi-type:
      "[Accommodation]": CRS01
      "[Education, Training and Employment (ETE)]": CRS02
      "[Finance, Benefits and Debt (FBD)]": CRS03
      "[Dependency and Recovery (D&R)]": CRS04